x-pack/plugins/cloud_security_posture/common/schemas/csp_rule.ts

```diff
@@ -5,27 +5,38 @@
  * 2.0.
  */
 import { schema as rt, TypeOf } from '@kbn/config-schema';
+import { cspRuleMetadataSchema } from './csp_rule_metadata';
 
-// TODO: needs to be shared with cloudbeat
-export const cspRuleSchema = rt.object({
+export const cspRuleSchemaV830 = rt.object({
+  audit: rt.string(),
+  benchmark: rt.object({ name: rt.string(), version: rt.string() }),
+  default_value: rt.nullable(rt.string()),
```

Comment:
So 8.3 ended up with `nullable` and not `maybe`? Is this aligned across our repos? I see that in 8.4 you used `maybe` instead...

Reply:
yes, in all our repos for 8.3 we are aligned with nullable.
And in 8.4 we move that to omit them instead.
makes sense?